guess_the_number.js

```diff
@@ -0,0 +1,41 @@
+import rl from 'readline-sync'
+
+const theTruth = Math.floor( Math.random() * 100 ) + 1
+
+const isValid = input => {
+  return  ( parseInt( input ) === true ) ||
+          ( parseInt(input) < 101 && parseInt(input) > 0 )
```

Comment:
Indentation is inconsistent.  There need only be one space between keyword `return` and the successive expression.  I would also avoid aligned indentation - it forces any successive developer to do the work of perfectly aligning in order to maintain this style.  A consistent two space indent is sufficient.

Minor style note - you may omit the parentheses around the first expression. `parseInt( input ) === true`